### ISSUE TITLE
Add support to create EN topics, destinations and subscriptions

### ISSUE THREAD

**Issue by ocofaigh** [terraform-ibm-modules/terraform-ibm-event-notifications#197]
IBM provider supports it, so should be exposed in this module. See https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-tera-workwith

Needs to be added to root level module, fscloud submodule(modules/fscloud) and the DA (solutions/standard)

**Comment by jor2** [comment]
@ocofaigh That link uses provider version `1.38.2`. It seems they've updated it since then, this is the list of possible destinations.

````
[ibm_en_destination_ce](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_ce)
[ibm_en_destination_cf](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_cf)
[ibm_en_destination_chrome](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_chrome)
[ibm_en_destination_cos](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_cos)
[ibm_en_destination_custom_email](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_custom_email)
[ibm_en_destination_custom_sms](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_custom_sms)
[ibm_en_destination_firefox](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_firefox)
[ibm_en_destination_huawei](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_huawei)
[ibm_en_destination_ios](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_ios)
[ibm_en_destination_msteams](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_msteams)
[ibm_en_destination_pagerduty](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_pagerduty)
[ibm_en_destination_safari](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_safari)
[ibm_en_destination_slack](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_slack)
[ibm_en_destination_sn](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_sn)
[ibm_en_destination_webhook](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/en_destination_webhook)
````
I'll add support for cos, is there other ones we would like too?

**Comment by ocofaigh** [comment]
@jor2 I'm not sure if we should actually add any of these to the module itself. Perhaps we could update one of the examples to show how they can be used directly with the provider, but otherwise we are just wrapping provider code in our module, and it doesn't give any extra benefit. I decreased the severity of this internally since we decided its actually the other DAs (Secrets Manager, SCC) themselves that should be creating these and not the EN module itself.
I'm going to close the issue.
